tests/unit/test_charm.py

```diff
@@ -1,7 +1,251 @@
-# Copyright 2023 Canonical
+# Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
 
-class TestCharm:
-    def test_dummy_pass(self):
-        assert True
+import tempfile
+import unittest
+from pathlib import Path
+from unittest.mock import Mock, patch
+
+from ops.model import ActiveStatus, BlockedStatus, WaitingStatus
+from scenario import Container, Context, Model, Mount, Relation, State  # type: ignore[import]
```

Comment:
why did you have to type:ignore this line? issues with types?

Reply:
Yes, we use `mypy`, and it complains that the package does not have library stubs or a `py.typed` marker. I think on the project you can simply add a `py.typed` file for the package to work with `mypy`.